# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/uclibc/strverscmp.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- uClibc-ng-1.0.39/libc/string/strverscmp.c.vanilla	2021-10-07 14:04:26.792087065 +0200
+++ uClibc-ng-1.0.39/libc/string/strverscmp.c	2021-10-07 14:04:31.454087398 +0200
@@ -103,4 +103,4 @@
       return state;
   }
 }
-libc_hidden_def(strverscmp)
+//libc_hidden_def(strverscmp)
--- uClibc-ng-1.0.39/include/string.h.vanilla	2021-10-07 14:04:26.796087066 +0200
+++ uClibc-ng-1.0.39/include/string.h	2021-10-07 14:05:04.975089788 +0200
@@ -478,7 +478,7 @@
 /* Compare S1 and S2 as strings holding name & indices/version numbers.  */
 extern int strverscmp (const char *__s1, const char *__s2)
      __THROW __attribute_pure__ __nonnull ((1, 2));
-libc_hidden_proto(strverscmp)
+//libc_hidden_proto(strverscmp)
 
 # if 0			/* uClibc does not support strfry or memfrob. */
 /* Sautee STRING briskly.  */
